addons/gamification/views/mail_templates.xml

```diff
@@ -20,7 +20,9 @@
                 <p>
                     You just reached a new rank : <strong>${object.rank_id.name}</strong>
                 </p>
+                % if object.next_rank_id.name
                 <p>Continue your work to become a <strong>${object.next_rank_id.name}</strong> !</p>
+                % endif
```

Comment:
Reading the code this feels like it is a bug fix?

Reply:
Kind of, but there is a line about it in the PAD. What should I do ?

Reply:
As for all fixes, whether or not they come from a pad, they should be done in the version in which the bug occurred.